api/proto/config/vm.proto

```diff
@@ -41,4 +41,9 @@ message VmConfig {
   uint32 vncDisplay = 17;
   string vncPasswd = 18;
   bool disableLogs = 19;
+  // Set if the VM should run in a mode with pinned CPUs. In this case all the
+  // VCPUs threads created by QEMU run on a dedicated physical CPU. All the
+  // other QEMU threads will be limited to run on the physical CPUs defined by
+  // CPU mask.
+  bool cpuPinned = 20;
```

Comment:
And in the same vein, "pinCpu" would sounds morelike a config statement.
(We currently have some  of the form "enableX" and many "X" which could be read as "setX", in which case we have an imperative verb followed by the object/setting".)

Also, if you put lower_snake_case in the proto file as yetus/buflint suggests, the protoc compiler will generate golang strucures in CamelCase.

Reply:
Ok, will do that now.